uiowa/uiowa
# [SASS] Transition uids_base SASS to using the module system instead of imports

Related to https://github.com/uiowa/uids/pull/954. 

# How to test

-  Code review
- `blt frontend` and confirm that the only log messages are from `sitenow_wysiwyg.scss`. 
-  Spot check a few sites and make sure that the styles work as expected. 

## Todo 
- [x] Create separate issue to change how styles are handled in `sitenow_wysiwyg.scss`
